src/utils/sdfg/check_and_redirect_edge.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+export function check_and_redirect_edge(edge, drawn_nodes, sdfg_state) {
```

Comment:
same here for sdfg utility functions. It's a bit too many files that have the same purpose.